app/constants/autocomplete.js

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+export const AT_MENTION_REGEX = /\B(@([^@\r\n\s]*))$/i;
```

Comment:
Small improvement here, you can get rid of the outer brackets and make this `/\B@([^@\r\n\s]*)$/i` to avoid having to do the `isSearch ? match[1] : match[2]` [here](https://github.com/mattermost/mattermost-mobile/pull/988/files#diff-d8e1907524aa62484d09c898587f4b91R29)